sql/init_data/init.sql

```diff
@@ -51,4 +51,14 @@ CREATE TABLE "A a" (col int primary key);
 -- test for issue #44 github
 CREATE VIRTUAL TABLE fts_table USING fts5(name, description, tokenize = porter);
 
+-- github.com/pull/59
```

Comment:
It is better if you update the comment to describe about the feature, so later user can understand the feature easily without reopen issue/pull request to check.
For example, add description "test for updatable option".

Reply:
Updated, please verify.

Reply:
Thanks for updating. I confirmed.